pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     'humanfriendly>=8.2',
     "numpy>=1.21.0;python_version<'3.10'",
     "numpy>=1.23.3;python_version>='3.10' and python_version<'3.12'",
-    "numpy>=1.26.0;python_version>='3.12'",
+    "numpy>=1.21.0;python_version>='3.12'",
```

Comment:
this is still wrong. There is no numpy 1.21.0 for python 3.12. We should leave this the same as it was since our minimum tests will attempt to install whatever is marked here